modules/jdbc-test/src/test/java/org/testcontainers/junit/SimplePostgreSQLTest.java

```diff
@@ -11,29 +11,68 @@
 import java.sql.Statement;
 
 import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertNotEquals;
 
 /**
  * @author richardnorth
  */
 public class SimplePostgreSQLTest {
+    @Test
+    public void testSimple() throws SQLException {
+        PostgreSQLContainer postgres = new PostgreSQLContainer();
```

Comment:
Let's wrap container creation in test-with-resources block, this way we can also omit the finally block.

```
try (PostgreSQLContainer postgres = new PostgreSQLContainer();) {
  postgres.start()
  // do the test, no call to stop needed
}
```